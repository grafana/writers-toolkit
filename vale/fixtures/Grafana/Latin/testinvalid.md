Don't use:

- e.g
- e.g.
- e.g,
- eg
- eg.
- eg,
- i.e
- i.e.
- i.e,
- ie
- ie.
- ie,
