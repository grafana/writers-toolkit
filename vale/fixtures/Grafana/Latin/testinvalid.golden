testinvalid.md:3:3:Grafana.Latin:Use 'for example' instead of 'e.g'.
testinvalid.md:4:3:Grafana.Latin:Use 'for example' instead of 'e.g'.
testinvalid.md:5:3:Grafana.Latin:Use 'for example' instead of 'e.g'.
testinvalid.md:6:3:Grafana.Latin:Use 'for example' instead of 'eg'.
testinvalid.md:7:3:Grafana.Latin:Use 'for example' instead of 'eg'.
testinvalid.md:8:3:Grafana.Latin:Use 'for example' instead of 'eg'.
testinvalid.md:9:3:Grafana.Latin:Use 'that is' instead of 'i.e'.
testinvalid.md:10:3:Grafana.Latin:Use 'that is' instead of 'i.e'.
testinvalid.md:11:3:Grafana.Latin:Use 'that is' instead of 'i.e'.
testinvalid.md:12:3:Grafana.Latin:Use 'that is' instead of 'ie'.
testinvalid.md:13:3:Grafana.Latin:Use 'that is' instead of 'ie'.
testinvalid.md:14:3:Grafana.Latin:Use 'that is' instead of 'ie'.
